usecases/user/login.md

```diff
@@ -73,15 +60,32 @@
             <strong>Result</strong>
         </td>
         <td>
-            The <a href="../user.md"> user </a> is logged in
+            The actor is logged in
         </td>
     </tr>
     <tr>
         <td>
             <strong>Exceptions</strong>
         </td>
         <td>
-            NA
+             <li>
+                if the everything goes right:
+                <ul>
+                    <li>
+                        if credentials are valid
+                        <ul>
+                            the system logs in the user
+                        </ul>
+                    </li>
+                    <li>
```

Comment:
unnecessary block as the "correct case" is covered by scenario. Just "else case" (exceptions) to be listed here

Reply:
everything else was fine